### DISCUSSION TITLE
Plugins

### DISCUSSION THREAD

**Discussion by norton120** [palmetto/palm-cli#51] [Show and tell]
What cool plugins are people making?

**Comment by norton120** [comment]
I started in on a git plugin here https://github.com/norton120/palm-git - seems like a no-brainer. If someone wants to publish it to pypi, my account is frozen while we unwind a lost 2FA device, so it'll be a minute before I can get that started. but this could be cool I think

**Reply by jakeberesford-palmetto** [reply]
@norton120 you could publish it under the palmetto pypi account. Creds are in the 1password vault

**Reply by norton120** [reply]
will do just gotta get to that computer :+1:
